webrender/src/picture.rs

```diff
@@ -520,8 +532,14 @@ impl TileCache {
                 let key = (px.round() as i32, py.round() as i32);
 
                 let mut tile = match old_tiles.remove(&key) {
-                    Some(tile) => tile,
-                    None => Tile::new(),
+                    Some(tile) => {
+                        tile
+                    }
```

Comment:
This indentation is weird - does rustfmt require it? I'd think that this could still just be => tile, without braces.

Reply:
Fixed.